keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -1029,7 +1023,9 @@ The conflict will be resolved by assigning precedence based on each
 A derived service will be accessible with the clusterset IP at the ports
 dictated by child services. If the external properties of service ports for a
 set of exported services don’t match, the clusterset service will expose the
-union of service ports declared on its constituent services.
+union of service ports declared on its constituent services and raise a `PortConflict`
+conflict condition. In that case, network traffic should be directed only to endpoints
+from constituent services that actually expose the destination port.
```

Comment:
Sorry not trying to nitpick but I don't think "destination port" is clear wrt the prior text which talks about the "union of service ports". I think I know what you're trying to say, how about:


```suggestion
conflict condition. In that case, network traffic to a conflicting port should only be directed to endpoints from constituent services that actually expose the port.
```